### PR TITLE
Create class AnimalRepo

### PR DESCRIPTION
**Pull request by JoergWolff**
Is this compatible?

### CODE REVIEW

**Review by ryliecc — APPROVED** [review]
nice 🥳 lgtm

File: src/main/java/de/wolffclan/Main.java [hunk 1]
```diff
@@ -1,10 +1,13 @@
...
 
+import de.wolffclan.animal.AnimalRepo;
+
 public class Main {
     public static void main(String[] args) {
 
         System.out.println("ffm-java-23-1-GitWorkflow \n ------------");
 
-
+        AnimalRepo animals = new AnimalRepo();
```
Comment (JoergWolff) [inline src/main/java/de/wolffclan/Main.java:10]: Its new

File: src/main/java/de/wolffclan/Main.java [hunk 2]
```diff
@@ -1,10 +1,13 @@
 package de.wolffclan;
 
+import de.wolffclan.animal.AnimalRepo;
+
 public class Main {
     public static void main(String[] args) {
 
         System.out.println("ffm-java-23-1-GitWorkflow \n ------------");
 
-
```
Comment (JoergWolff) [inline src/main/java/de/wolffclan/Main.java:8]: Unneccessary space

**Review by KB-HH — COMMENTED** [review]
super

**Pull request merged**